pyproject.toml

```diff
@@ -1,21 +1,23 @@
+[build-system]
+requires = ["poetry-core"]
+build-backend = "poetry.core.masonry.api"
+
 [tool.poetry]
 name = "cfret-screen-map-analysis"
 version = "0.0.1"
 description = "Analysis notebooks applying mean average Precision (mAP) to the CFReT screen image-based profiles dataset."
 authors = ["axiomcura <erikishere3@gmail.com>"]
 readme = "README.md"
+packages = [
+  {include = "src"} # including analysis module
```

Comment:
Double checking on the comment here: is this an analysis or utility module? Consider removing the comment to help make this agnostic to change in the `src` dir.

Reply:
This is a utility module where it contains functions that are used in the analysis.